services/rook-ceph-cluster/1.10.8/rook-ceph-cluster.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   dependsOn:
     # There can only be one operator per cluster, just ensure CRDs are present, no need to wait for actual operator Deployment.
-    - name: rook-ceph-cluster-pre-install
+    - name: rook-ceph-cluster-pre-install-v1.10.8
```

Comment:
cc @bcmendoza @gracedo @sonofgibs idk if this can be optimized in the automated chartbumps but at the least this is something to be taken care for all future `rook-ceph-cluster` chart bumps

Reply:
included in https://d2iq.atlassian.net/browse/D2IQ-95824